src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -45,7 +44,7 @@ module li_calving
    !
    !--------------------------------------------------------------------
 
-   public :: li_calve_ice, li_restore_calving_front
+   public :: li_calve_ice, li_restore_calving_front, apply_front_ablation_velocity
```

Comment:
The MPAS convention is to proceed all public routine with `li_`, so now that this routine is public, that should be added everywhere.

Reply:
li_ prefix added everywhere